.github/workflows/validate-codeowners.yml

```
@@ -0,0 +1,21 @@
+name: LocalStack - Validate Codeowners
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  validate-codeowners:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Validate codeowners
+        uses: mszostok/codeowners-validator@v0.7.4
+        with:
+          checks: "files"
```

I think it would be nice to also enable the `duppatterns`, the `syntax`, and the (experimental, but well working) `avoid-shadowing` check. The `owners` check requires a github token, but I think it's not important enough.
```suggestion
        with:
          checks: "files,duppatterns,syntax"
          experimental_checks: "avoid-shadowing"
```

---

Thanks for raising this comment, I wasn't aware of these features!
I have adjusted the workflow to include the addition checks, seems like we're good to go as everything has passed 🚀